src/classes/frSetupController.cls

```diff
@@ -104,12 +104,7 @@ public with sharing class frSetupController {
 		allMappings.addAll(donorMappings);
 		for(frMapping__c mapping : allMappings) {
 			if(String.isNotBlank(mapping.sf_Name__c)) {
-				if(mapping.Is_Constant__c) {
-					mapping.Name = mapping.Constant_Value__c;
-				} else {
-					mapping.Name = mapping.fr_Name__c;
-				}
-				mapping.Name += ' -> ' + mapping.sf_Name__c;
+				mapping.Name = (mapping.Type__c == DONATION_TYPE ? 'o' : 'c') + mapping.sf_Name__c;
```

Comment:
Can you add a '.' after the single letter prefix? I think it's currently possible to have some fields that differ only by one letter which end up giving the same name (if I have 2 fields that both start with a `c`, and are entirely the same except one has 1 more `c` at the start or something?)

Reply:
Updated, new beta package link:
https://login.salesforce.com/packaging/installPackage.apexp?p0=04t15000000OBuN